packages/protocol/contracts/stability/Reserve.sol

```diff
@@ -149,17 +222,29 @@ contract Reserve is IReserve, Ownable, Initializable, UsingRegistry, ReentrancyG
     return _tokens;
   }
 
+  function getCustodians() external view returns (address[] memory) {
+    return _custodians;
+  }
+
+  function getReserveGoldBalance() public view returns (uint256) {
```

Comment:
Nice! We'll want to use this in `EpochRewards.sol` when calculating the voting gold percentage

Reply:
Cool!